composables/massmint/useMassMint.ts

```diff
@@ -34,48 +34,67 @@ export const statusClass = (status?: Status) => {
 }
 
 export const useCollectionForMint = () => {
-  const collectionsEntites = ref<MintedCollection[]>()
+  const collections = ref<MintedCollection[]>()
   const { accountId } = useAuth()
   const { urlPrefix } = usePrefix()
 
   const { data, isPending } = useQuery({
     queryKey: ['collections-for-mint', accountId, urlPrefix],
+    queryFn: async () => {
+      if (!accountId.value) {
+        return null
+      }
+
+      collections.value = []
+
+      return (await useAsyncGraphql({
+        query: 'collectionForMint',
+        variables: {
+          account: accountId.value,
+        },
+      })).data.value
+    },
+  })
+
+  const collectionEntities = computed(() => data.value?.collectionEntities || [])
+
+  const { data: collectionTotalCounts } = useQuery({
+    queryKey: ['collection-total-counts', computed(() => collectionEntities.value?.map(c => c.id))],
     queryFn: async () =>
-      accountId.value
-        ? (await useAsyncGraphql({
-            query: 'collectionForMint',
-            variables: {
-              account: accountId.value,
-            },
-          })).data.value
+      collectionEntities.value?.length
+        ? await Promise.all(collectionEntities.value.map(collection => new Promise((resolve, reject) =>
+          fetchGraphql(`
+            nftEntitiesConnection(
+              orderBy: blockNumber_ASC,
+              where: {
+                collection: { id_eq: "${collection.id}" }
+                burned_eq: false
+            }) {
+                totalCount
+            }
+          `)
+            .then(response => resolve(response.data.nftEntitiesConnection.totalCount))
+            .catch(reject),
+        )))
         : null,
```

Comment:
I would rather write it as a block 

```js
if (!collectionEntities.value?.length) {
  return null
}

const ...
```

Moreover you know how many nfts was burned :) 

cause `supply` will change if you burn something 
but `nftCount` not

Reply:
thanks! wasn't aware of that field , updated 87ebd22